src/parse_args.ts

```diff
@@ -104,6 +111,20 @@ program
     '--quiet-exit-code',
     'always return 0 as an exit code status, regardless of test pass / fail. Only return > 0 exit codes on internal errors where the suite could not be run'
   )
+  .addOption(
+    new Option('--download-throughput <throughput>', 'Maximal aggregated download throughput (bytes/sec). -1 disables download throttling.')
+      .default(defaultNetworkConditions.downloadThroughput)
+  )
+  .addOption(
+    new Option('--upload-throughput <throughput>', 'Maximal aggregated upload throughput (bytes/sec). -1 disables upload throttling.')
+      .default(defaultNetworkConditions.uploadThroughput)
+  )
+  .addOption(
+    new Option('--latency <latency>', 'Minimum latency from request sent to response headers received (ms).')
+      .default(defaultNetworkConditions.latency)
+  )
+  .option('--offline', 'True to emulate internet disconnection.', defaultNetworkConditions.offline)
```

Comment:
Is this to disable the internet or to disable all network connections? The way this is written it sounds like the LAN would still work. I'm pretty sure that's not the case, so we should clarify and `s/internet/network`

Reply:
You're right, I've updated it. Interesting enough, I pulled the descriptions straight from the devtools docs

https://chromedevtools.github.io/devtools-protocol/tot/Network/#method-emulateNetworkConditions

Reply:
Can we kill this offline flag for now and add this later if we have a new feature request?

Reply:
@vigneshshanmugam can you share your reasoning for removing this? I'm ++ on removing it myself because it's beyond the scope of the original issue, and I think we shouldn't add options unless we have a clear use-case behind it.

The only use case I can think of is testing PWAs, but then maybe that should be on a per-journey basis? Either way, I think it could use some refinement

Reply:
Apologies for not posting about the case, Yes its a pretty nifty use-case when it comes to testing web apps and only applies to Web apps that uses Service workers (PWA). We can introduce this flag if we ever need to support that use-case in the future and also its not in the scope of this PR.